src/lib/pubkey/ec_group/ec_apoint.cpp

```diff
@@ -140,6 +149,19 @@ EC_AffinePoint EC_AffinePoint::negate() const {
    return EC_AffinePoint(std::move(pt));
 }
 
+std::vector<uint8_t> EC_AffinePoint::serialize(EC_Point_Format format) const {
+   if(format == EC_Point_Format::Compressed) {
+      return this->serialize_compressed();
+   } else if(format == EC_Point_Format::Uncompressed) {
+      return this->serialize_uncompressed();
+   } else {
+      // TODO(Botan4) Remove this
+      auto enc = this->serialize_uncompressed();
+      enc[0] ^= enc[enc.size() - 1] & 0x01;
```

Comment:
I'm guessing this implements the hybrid encoding? Perhaps a comment would be helpful, for clarity. :)

Reply:
Added a comment